specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,571 @@
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.6); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"15/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+    data.SequenceNumber(1); // The sequence number has to be incremental for the platform to render progress
+
+    toast.ProgressData(data);
+    ToastNotificationManager::ShowToast(toast);
+}
+
+// Send subsequent progress updates
+winrt::Windows::Foundation::IAsyncAction UpdateProgressAsync()
+{
+    winrt::hstring tag = L"weekly-playlist";
+    winrt::hstring group = L"downloads";
+
+    // Assign new values
+    // Note that you only need to assign values that changed. In this example we don't assign progressStatus since we don't need to change it
+    ToastProgressData data = winrt::make<ToastProgressData>();
+    data.Value(0.7); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"18/26 songs"); // Binds to {progressValueString} in xml payload
+    data.SequenceNumber(2);
+
+    auto result = co_await ToastNotificationManager::UpdateToastProgressDataAsync(data, tag, group);
+    if (result != ToastProgressResult::Succeeded)
+    {
+        LOG_HR_MSG(E_UNEXPECTED, "Toast Progress Update Failed!");
+    }
+}
+```
+
+# Remarks
+
+## Registration
+
+The developer should always call the Toast Registration API first to register the current process as the Activator target.
+
+## Foreground API calls
+
+The developer should always subscribe to toast activation foreground events to intercept toasts if the app happens to be running in the foreground.
+
+## Manifest Registration
+
+For MSIX, the COM activator GUID and the exe need to be registered in the manifest. The launch args
+would need to be pre-set to a well-known string that defines Toast Activation Triggers.
+
+```xml
+<Extensions>
+  <com:Extension Category="windows.comServer">
+    <com:ComServer>
+      <com:ExeServer Executable="SampleBackgroundApp\SampleBackgroundApp.exe" DisplayName="SampleBackgroundApp" Arguments="----ToastActivationServer">
+        <com:Class Id="{GUIDEntryPoint}" />
+      </com:ExeServer>
+    </com:ComServer>
+  </com:Extension>
+</Extensions>
+```
+
+# API Details
+
+```c#
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Windows.ToastNotifications
+{
+    [experimental]
+    // The Shell asset details for Unpackaged App Registrations 
+    runtimeclass ToastAssets
+    {
+        // Initialize using Shell assets like DisplayName, iconPath and icon Background color
+        ToastAssets(String displayName, String iconPath, Windows.UI.Colors iconBackgroundColor);
+
+        // Initialize using Shell assets like DisplayName and iconPath
```

Comment:
@nicuparente Did we close on this with the Shell folks yet?

Reply:
For now, I went ahead and removed the overload that takes in the color. I believe we are going with the theme color for the device as the POR unless we are told otherwise.